### DISCUSSION TITLE
How to login Github to clone repositories?

### DISCUSSION THREAD

**Discussion by writingdeveloper** [coder/code-server#2429] [Q&A] [answered]
I already checked the #1883 and It works before. But I reinstalled the code-server. And do the same process, it's not working.

What is the difference Add Github Token and Github (Extension) Login.

But actually, both token & extension function is not working.

And even when I use both method, profile icon that located left below said "Not yet login"

Where is the problem come from. Is there anyone experience same situation?

**Comment by code-asher** [comment]
Nothing works for me either. I'm currently working on fixing this so
hopefully it should be working soon.

**Comment by writingdeveloper** [comment, marked as answer]
In v3.8.0 this bug is solved! 
Upgrade the code-server

**Reply by rifat-h** [reply]
How do I Upgrade code-server ? any documentation link ?

**Reply by lg-nineoneninethreenine** [reply]
Just use the installation cURL line, code-server will be updated.
